awsshell/utils.py

```diff
@@ -142,3 +144,8 @@ def force_unicode(obj, encoding='utf8'):
         if not isinstance(obj, six.text_type):
             obj = _attempt_decode(obj, encoding)
     return obj
+
+
+def format_json(response):
```

Comment:
It was being implicitly tested through one of the Environment's unit tests, but I agree an explicit test for this too makes sense.